eng/pipelines/prettier.yml

```diff
@@ -0,0 +1,14 @@
+trigger: none
+
+jobs:
+- job: Prettier
+  pool:
+    name: azsdk-pool-mms-ubuntu-2204-general
+    vmImage: ubuntu-22.04
+
+  steps:
+  - template: /eng/pipelines/templates/steps/npm-install.yml
+
+  - pwsh: |
+      $(Build.SourcesDirectory)/eng/scripts/Swagger-Prettier.ps1 -Verbose
+    displayName: Swagger Prettier Check
```

Comment:
Same as I commented here:
- https://github.com/Azure/azure-rest-api-specs/pull/25718/files/71ec626a37812322300b7f15bc1ede2bde75cc53#r1320392877

Maybe `OpenAPI Prettier Valiation`

Reply:
For the pipeline name I'm thinking we should just call it "Prettier" so that will be the name of the check but and Swagger/OpenApi is just one set of prettier checks that we may end up wanting. However if we don't think we will check any other file types then we can make this remain specific to openapi and call the pipeline "Validate OpenApi Prettier" or "OpenApi Prettier Check" seems more natural.

Reply:
Yes that is the template that gets used they also include Stage name if there is one. I was trying to figure out how to customize so I could put all the checks in one pipeline with different jobs but I was unable to make that work.

@mikeharder so you vote would be to call this "Swagger Prettier Check" as opposed to renaming it to "OpenAPI Prettier Check"?

Reply:
I don't feel strongly about the specific check names if you want to start renaming them to be more consistent or use a new pattern.  But I think we should either keep the check names consistent for now to minimize confusion, or do a big rename at once.  I think it would be bad for spec authors and reviewers if the names are changing at random times.

Reply:
I agree I'll keep the "Swagger PrettierCheck" name until we finish with the larger transition.